generate.jl

```diff
@@ -0,0 +1,27 @@
+using Pkg
+Pkg.activate(".")
+using Fire
+using PhysicsTutorials
+
+path_names(path::String) = path in ("","/") ? () : (path_names(dirname(path))..., basename(path))
+
+@main function generate(filename::String)
+    pns = path_names(abspath(filename))
+    length(pns) < 3 && error("illegal path $filename")
```

Comment:
"Illegal" sounds a bit strong. Maybe replace by something like `Invalid path to source file. Expected format: "tutorials/<category>/<tutorial_name>/<tutorial_name>.ipynb (or .jl/.jmd)"`?